tests/14account/01change-password.pl

```diff
@@ -0,0 +1,220 @@
+my $password = "my secure password";
+
+
+test "After changing password, can't log in with old password",
+   requires => [ local_user_fixture( password => $password ) ],
+
+   check => sub {
+      my ( $user, ) = @_;
+
+      do_request_json_for( $user,
+         method => "POST",
+         uri    => "/r0/account/password",
+         content => {
+            auth => {
+               type => "m.login.password",
+               user     => $user->user_id,
+               password => $password,
+            },
+            new_password => "my new password",
+         },
+      )->then( sub {
+         do_request_json_for( $user,
+            method => "POST",
+            uri    => "/r0/login",
+
+            content => {
+               type     => "m.login.password",
+               user     => $user->user_id,
+               password => $password,
+            }
+         # We don't mandate the exact failure code here
+         # (that should be done in the login test if
+         # anywhere), any 4xx code is fine as far as
+         # this test is concerned.
+         )->main::expect_http_4xx;
+      }
+      )->then_done(1);
```

Comment:
Fold the above two lines together so as not to have the awkward "two closing lines with same indent" thing.